app/routes/socialEvents.tsx

```diff
@@ -0,0 +1,41 @@
+import EventCard from '~/components/common/eventCard';
+import moment from 'moment';
+
+const startTime = moment().format('h:mm A');
+const endTime = moment().add(2, 'hours').format('h:mm A');
+
+export default function socialEvents() {
+  const dummyEvent = {
```

Comment:
Could you create a ticket to ensure we have a file to store these mock data

Reply:
Please refer: https://github.com/Real-Dev-Squad/website-calendar/issues/136 @harshith-venkatesh